### DISCUSSION TITLE
Cables offline/local version

### DISCUSSION THREAD

**Discussion by constructor** [cables-gl/cables_docs#689] [Answers & Questions] [answered]
I have watched this video: 

> Cables development preview: standalone / offline version WIP
> https://www.youtube.com/watch?v=G6QFFDCYmn4

I see that the cables standalone seems to use electron to build the 'native' app. 
Has there been any consideration given to using docker so users can run local instances of cables?
Being able to 'docker compose up' a cables instance would be simple.

**Comment by steam0r** [comment, marked as answer]
we will provide a way to run and develop cables locally one day, this will also be based on electron, as you would need a browser anyhow. no need for a webserver or database or anything that docker provides.

**Comment by constructor** [comment]
OK. I just thought it would be simpler, but then again, for non developers it would probably not be.
